Release-Notes-R6.md

```diff
@@ -1,8 +1,134 @@
+
+## Updated software
+
+| Software       | Version  |
+|----------------|----------|
+| flux2          | v2.2.3   |
+| sonobuoy       | v0.57.1  |
+| Cilium         | v1.15.1  |
+| Cilium cli     | v0.15.23 |
+| Cilium Hubble  | v0.13.0  |
+| cert-manager   | v1.14.2  |
+| helm           | v3.14.1  |
+| metrics-server | v0.7.0   |
+| nginx-ingress  | v1.9.6   |
+| k9s            | v0.31.9  |
+| calico         | v3.27.2  |
+| capi           | v1.6.2   |
+| capo           | v0.9.0   |
+
+### k8s versions (1.25 -- 1.28)
+
+We test Kubernetes versions 1.25 -- 1.28 with the R6 Cluster API
+solution. We had tested earlier versions (down to 1.21) successfully before,
+and we don't expect them to break, but these are no longer supported
+upstream and no fresh node images are provided by us.
+
+Release notes for upstream Kubernetes can be found [here](https://github.com/kubernetes/kubernetes/releases).
+Please read the [API deprecation notes](https://kubernetes.io/docs/reference/using-api/deprecation-guide/#v1-29)
+when you move your workloads to the latest k8s versions.
+
+Kubernetes v1.29 can be deployed as a technical preview for now, but
+we expect that it will be stabilized soon.
+
+## New features
+
+### OpenTofu
+
+In R5 we protected users from accidentally using unfree BSL licensed code by terraform version constraint.
+Now, in [#606](https://github.com/SovereignCloudStack/k8s-cluster-api-provider/pull/606), we replaced Terraform
+with [OpenTofu](https://opentofu.org/),
+an open-source, community-driven IaC tool. Users don't have to worry anymore.
+
+### ClusterClass
+
+From [#600](https://github.com/SovereignCloudStack/k8s-cluster-api-provider/pull/600), this repository uses CAPI
+[ClusterClass](https://cluster-api.sigs.k8s.io/tasks/experimental-features/cluster-class/) feature for the creation of
+clusters, see k8s [blog](https://kubernetes.io/blog/2021/10/08/capi-clusterclass-and-managed-topologies/) for overview.
+This feature is also used in the SCS [Cluster Stacks](https://github.com/SovereignCloudStack/cluster-stacks) - KaaS
+reference implementation v2.
+
+### Proxy
+
+[#418](https://github.com/SovereignCloudStack/k8s-cluster-api-provider/issues/418) is about support for corporate proxy
+in the Kubernetes clusters. In R6, users are able to specify e.g. `http_proxy = "http://10.10.10.10:3128"`
+and this proxy setting will be propagated to the management host as well as on the worker and control plane nodes,
+see [#620](https://github.com/SovereignCloudStack/k8s-cluster-api-provider/pull/620)
+and [#645](https://github.com/SovereignCloudStack/k8s-cluster-api-provider/pull/645) for details.
+There is also a `no_proxy` setting for configuring exceptions,
+[#651](https://github.com/SovereignCloudStack/k8s-cluster-api-provider/pull/651).
+
+### OVN LB
+
+From R4, LoadBalancer in front of ingress-nginx can be configured to use the OVN provider
+instead of the default Amphora provider. But, it was only a tech preview feature and not recommended.
```

Comment:
Consider reformulation, e.g.: Starting from R4, it's possible to configure the LoadBalancer in front of ingress-nginx to utilize the OVN provider instead of the default Amphora provider. However, it's important to note that this capability was introduced as a **tech preview** feature and was not recommended for production use.

Reply:
Thank you, please see 90ca44f5cfa538c7aa292f4c2069e793c140412e